src/kata5.reachDestination.js

```diff
@@ -1,3 +1,7 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  const time = Math.round((2 * distance) / speed) / 2;
```

Comment:
Nice. I would probably use ceil here if you always want to round up the hours but I think it doesn't matter as you are doing the multiplication and division. your answer is still correct. :)